lib/src/Base/Func/openturns/Evaluation.hxx

```diff
@@ -159,6 +159,24 @@ public:
                      const Indices & pointNumber = Indices(2, ResourceMap::GetAsUnsignedInteger("Evaluation-DefaultPointNumber")),
                      const GraphImplementation::LogScale scale = GraphImplementation::NONE) const;
 
+  /** Draw cross sections of the function assumed to have 1D output */
+  virtual GridLayout drawCrossCuts(const Point& centralPoint,
+    const Point& xMin,
+    const Point& xMax,
+    const Indices& pointNumber,
+    const Scalar vMin,
+    const Scalar vMax,
+    const Bool withMonoDimCuts = ResourceMap::GetAsBool("CrossCuts-DefaultWithMonoDimCuts"),
+    const Bool isFilled = ResourceMap::GetAsBool("Contour-DefaultIsFilled")) const;
+
+  /** Draw cross sections of the function assumed to have 1D output (vMin and vMax are calculated) */
+  virtual GridLayout drawCrossCuts(const Point& centralPoint,
+    const Point& xMin,
+    const Point& xMax,
+    const Indices& pointNumber,
+    const Bool withMonoDimCuts = ResourceMap::GetAsBool("CrossCuts-DefaultWithMonoDimCuts"),
+    const Bool isFilled = ResourceMap::GetAsBool("Contour-DefaultIsFilled")) const;
+
```

Comment:
I'd rather not have two methods with such close signatures which is ambiguous: could we keep the first one but with default values for vmin/vmax (with a different ordering maybe) ? or have different names for the methods ?

Reply:
@jschueller I removed one of the two methods and changed the order of the arguments.